src/lib/hash-input.unit-test.ts

```diff
@@ -201,3 +146,8 @@ function inputToOcaml({
     packed: packed.map(([field, size]) => ({ field, size })),
   };
 }
+
+function fixVerificationKey(accountUpdate: Json.AccountUpdate) {
+  // TODO we set vk to null since we can't generate a valid random one
+  accountUpdate.body.update.verificationKey = null;
```

Comment:
this doesn't necessarily refer to a null vk, but to an account update which doesn't change the verification key.
the test here just checks hashing of account updates.
testing that verification fails when an account has no verification key is more in scope for the Mina repo (since verification happens on-chain / snarkyjs isn't really concerned with verification)